src/Command/Serve.php

```diff
@@ -79,6 +94,8 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+        $io->title('Yii 3 Development Server.');
+        $io->writeln('https://yiiframework.com');
```

Comment:
```suggestion
        $io->writeln('https://yiiframework.com'."\n");
```

Reply:
It is `writeln` already. Do you mean additional line?

Reply:
Yes, here:
![image](https://user-images.githubusercontent.com/525501/235903252-07a7d170-9009-49f6-bbdf-3c002ea98635.png)